app/controllers/questions_controller.rb

```diff
@@ -2,7 +2,7 @@ class QuestionsController < ApplicationController
   before_action :assign_question, only: %i(show edit update destroy)
 
   def index
-    @questions = Question.all
+    @questions = Question.all.decorate
```

Comment:
This is the correct way to decorate a single object, not a collection.
Here is my suggestions:

- First: update the questions_controller_spec to ensure `@questions.first` is properly decorated. You can do that by expecting the first question of the array respond_to a Decorator's method. This test should fail right now, because of the current implementation.

- Second: read the Draper documentation, section "Collections" to know how to decorate a collection of records: https://github.com/drapergem/draper#decorating-objects

This is a nice use case of TDD! :)